_data/events_gdcr2018/Sydney.json

```diff
@@ -0,0 +1,20 @@
+{
+  "title": "Sydney Global Day of Code Retreat 2018",
+  "url": "https://www.eventbrite.com.au/e/sydney-global-day-of-code-retreat-2018-tickets-51246992133",
+  "moderators": [
+    "Thomas Eizinger",
+    "Lloyd Fournier",
+    "Philipp Hoenisch",
+    "Franck Royer",
```

Comment:
Please remove the `,`character here, so that the build can compile.

Then, we merge :-) .

Reply:
Ah, I guess too much Rust in last months :D